x-pack/plugins/security_solution/public/common/mock/endpoint/app_context_render.tsx

```diff
@@ -101,17 +104,16 @@ export type WaitForReactHookState =
     >
   | false;
 
-type HookRendererFunction<TProps, TResult> = (props: TProps) => TResult;
+type HookRendererFunction<TResult, TProps> = (props: TProps) => TResult;
```

Comment:
what's the reason for changing the order of these types?

Reply:
testing-library/react switched around the arguments for the generics provided on the  renderHook util,  I decided to propagate the same change here too